packages/notte-sdk/src/notte_sdk/endpoints/sessions.py

```
@@ -597,7 +613,7 @@ def scrape(self, **data: Unpack[ScrapeRequestDict]) -> DataSpace:
     def observe(self, **data: Unpack[ObserveRequestDict]) -> Observation:
         return self.client.page.observe(session_id=self.session_id, **data)
 
-    def step(self, **data: Unpack[StepRequestDict]) -> Observation:
+    def step(self, **data: Unpack[StepRequestDict]) -> StepResponse:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)